<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutFontaine</class>
 <widget class="QDialog" name="AjoutFontaine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutFontaine</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de la borne fontaine:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant voie publique:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom topographique:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrondissement:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_voiepublique">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_topographique">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_ville">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_arrondissement">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_confirmer">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>657</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="case_identifiant">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="case_longitude">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="case_latitude">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QPushButton" name="abandonner_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>720</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Abandonner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_confirmer</sender>
   <signal>clicked()</signal>
   <receiver>AjoutFontaine</receiver>
   <slot>valider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>672</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abandonner_btn</sender>
   <signal>clicked()</signal>
   <receiver>AjoutFontaine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>735</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valider()</slot>
 </slots>
</ui>
